api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionEmailAddress.java

```diff
@@ -15,51 +15,58 @@
 @Table(name = "institution_email_address")
 public class DbInstitutionEmailAddress {
 
-  @Id
-  @GeneratedValue(strategy = GenerationType.IDENTITY)
-  @Column(name = "institution_email_address_id")
   private long institutionEmailAddressId;
-
-  @ManyToOne(fetch = FetchType.LAZY)
-  @JoinColumn(name = "institution_id")
   private DbInstitution institution;
-
-  @Column(name = "email_address", nullable = false)
   private String emailAddress;
 
   public DbInstitutionEmailAddress() {}
 
-  public DbInstitutionEmailAddress(DbInstitution institution, String emailAddress) {
-    this.institution = institution;
-    this.emailAddress = emailAddress;
+  @Id
```

Comment:
I think you should only define entity objects where it makes sense to your application to use as a standalone object. As in something you think of. If it's just a table row (with no more real abstraction), then you may be able to absorb it into the containing entity.

Reply:
How would we specify which data table to use for that?